packages/jaeger-ui/index.html

```diff
@@ -14,6 +14,18 @@
     <base href="/" data-inject-target="BASE_URL" />
     <link rel="shortcut icon" href="/favicon.ico">
     <title>Jaeger UI</title>
+
+    <!--Paste your Google tag ID in the code sample below-->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=TAG_ID" >
+    </script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'TAG_ID')
+    </script>
```

Comment:
this is now how we use GA, please see the discussion in the ticket.

Reply:
In this way GA4 analytics support without any 3rd party libraries.

Reply:
yes, for tracking different pages of a static website, but not of a Reach app that is a single page.